src/controller/securemgr/authorizer/authorizer.go

```diff
@@ -0,0 +1,141 @@
+/*******************************************************************************
+* Copyright 2020 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*******************************************************************************/
+package authorizer
+
+import (
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/casbin/casbin"
+)
+
+// AuthorizerImpl structure
+type AuthorizerImpl struct{}
+
+const (
+	rbacPolicyFileName = "policy.csv"
+	policy_template    = "p, admin, /*, *\n" +
```

Comment:
Why don't we put the policy and model templates in the samples/rbac folder instead of template code?

Reply:
Please, see answer above

Reply:
Resolved.